docs/docs-beta/docs/dagster-plus/deployment/environment-variables/built-in.md

```diff
@@ -5,4 +5,33 @@ sidebar_position: 3
 sidebar_label: "Built-in variables"
 ---
 
-# Built-in environment variables
\ No newline at end of file
+
+[Dagster+](/todo) provides a set of built-in, automatically populated environment variables, such as the name of a deployment or details about a branch deployment commit, that can be used to modify behavior based on environment.
+
+### All deployment variables
+
+The following variables are available in every deployment of your Dagster+ instance, including full (e.g., `prod`) and branch deployments.
+
+| Key | Value |
+|---|---|
+| `DAGSTER_CLOUD_DEPLOYMENT_NAME` | The name of the Dagster+ deployment. <br/><br/>  **Example:** `prod`. |
```

Comment:
Thoughts on formatting the variable names as `code` vs keeping them in plain text? I can't decide which one i prefer. I see pete went with `code` for the metadata table https://github.com/dagster-io/dagster/blob/03f8cd0fb0bea478cced1e3f7211b90c0987f9b0/docs/docs-beta/docs/guides/data-modeling/metadata.md#L101

Reply:
my thoughts:
- having ALL_CAPS_TEXT makes me feel yelled at and looks less polished. When it's in `code` it feels softer and more intentional. Point to wrapping in `code`
- `code` formatting is harder to read in dark mode. point to not wrapping

Reply:
I like code! We can likely fix formatting in dark mode by asking Josh very kindly